pkg/ui/app/components/summaryBar.tsx

```diff
@@ -8,14 +8,11 @@ interface SummaryStatProps {
   format?: (i: number) => string;
 }
 
-function computeValue(value: number, format?: (i: number) => string): string {
+function computeValue(value: number, format: (i: number) => any = _.identity): any {
```

Comment:
Oh I guess this could just be `_.toString` instead? To avoid the `any`s? Maybe the previous version was better.